docs/source/conf.py

```diff
@@ -60,14 +59,20 @@
 napoleon_numpy_docstring = False
 napoleon_google_docstring = True
 
+
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 #
-# source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = {
+    '.rst': 'restructuredtext',
+    '.txt': 'markdown',
+    '.md': 'markdown',
+}
+
+autodoc_typehints = 'signature'
```

Comment:
What is the effect of this?

Reply:
Nothing as such since this is default effect.
If we update sphinx we can use other params which might make autodoc better.

[Here](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_typehints) is description

Reply:
the let's remove please :)